recipes/icechunk/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "icechunk" %}
+{% set version = "0.1.0-alpha.10" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/earth-mover/icechunk/archive/refs/tags/icechunk-v{{ version }}.tar.gz 
```

Comment:
@ocefpaf, the following results in an error

```bash
                      ^^^^^^^^^^^^^^
  File "/opt/conda/lib/python3.12/site-packages/conda_build/render.py", line 981, in render_recipe
    m.config.compute_build_id(m.name(), m.version(), reset=reset_build_id)
                                        ^^^^^^^^^^^
  File "/opt/conda/lib/python3.12/site-packages/conda_build/metadata.py", line 1533, in version
    check_bad_chrs(version, "package/version")
  File "/opt/conda/lib/python3.12/site-packages/conda_build/metadata.py", line 776, in check_bad_chrs
    raise CondaBuildUserError(
conda_build.exceptions.CondaBuildUserError: Bad character(s) (-) in package/version: 0.1.0-alpha.10.
```

do you have any recommendations on how to avoid this issue?

Reply:
Is that version in a valid PEP440 form? Also, you will need to follow CFEP-05 here or release/ask a stable version upstream.

Reply:
>  you will need to follow CFEP-05 here or release/ask a stable version upstream.

it's going to be a while before there's a stable release upstream, and i wanted to be able to access the same alpha versions that are currently available on PyPI via conda.

Reply:
> Is that version in a valid PEP440 form? 

i found a workaround for the version issue